cura/Settings/UserProfilesModel.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, parent = None):
     def _fetchInstanceContainers(self):
         global_container_stack = Application.getInstance().getGlobalContainerStack()
         if not global_container_stack:
-            return []
+            return {}, {}

```

Comment:
Document what and why two dicts.

Reply:
The return value is documented in the inherited documentation. I've documented the other return statement here: 097cc72d89882b0bb05fb2627f91f5d36dbf7042